### ISSUE TITLE
Componente Button: ambiguità per la presenza di nome accessibile "link" e ambiguità con i testi semplici.

### ISSUE THREAD

**Issue by cfabry** [italia/bootstrap-italia#1259]
### Esiste già una discussione sul tema che ti interessa, o su un tema simile?

- [X] Ho verificato e non esiste

### Versione della libreria

2.10

### Cosa
[https://italia.github.io/bootstrap-italia/docs/componenti/buttons/](url)

1. Nel primo esempio, ma poi anche negli altri e anche su Designers Italia, abbiamo l'elemento `<button>` con nome accessibile "link". Questo crea confusione tra i due componenti.
2. Negli esempi base (anche su Designers) per la classe .btn abbiamo un elemento senza alcuna distinzione visiva con un testo semplice.

### Perché

1. Il valore semantico di link e button è completamente diverso quindi opportuno eliminare ovunque la parola "link".
2. Immagino sia necessario per avere un elemento molto lineare da inserire in altri contenitori, esempio in liste di pulsanti già separati da elementi `<li>`, tuttavia può risultare ambiguo e indurre gli sviluppatori a creare elementi `<button>` indistinguibili dai testi normali (errore di accessibilità molto frequente).
Valutare se inserire un alert per questo problema, o, magari, valutare se possibile una soluzione alternativa che non permetta ai pulsanti di essere confusi con il testo.

**Comment by Fupete** [comment]
@cfabry concordo che come minimo la variante "Link" dovrebbe avere un testo "Link azione" più chiaro (cose che succedono in pagina) e poi essere del colore del link (blu nel ns caso) e sottolineata già di base. 

Anche io ho da sempre dubbi sull'utilità di questa variante che non sembra un pulsante... Però, oltre a componenti composti, in effetti può essere utile forse a quelle situazioni in cui l'azione cambia cose in pagina ma non deve necessariamente sembrare un pulsante. Penso a un "Leggi di più" che apre un collapse di testo ulteriore per esempio a seguire... 

Sarebbe utile forse sì anche ripensare il contenuto del callout accessibilità che già c'è in documentazione se non è abbastanza chiaro. Ora è così: 

![Image](https://github.com/user-attachments/assets/1a00f644-6eca-4ee2-b73c-884d756f7e03)

Metto in CC @zetareticoli che sta facendo il lavoro sui Design Tokens dove si potrebbe sistemare i colori intanto.

**Comment by zetareticoli** [comment]
Ho aperto la PR #1392 con anteprima delle modifiche alla documentazione e al codice. In sostanza, la sola classe `.btn` non può essere utilizzata per conferire colore e aspetto di un pulsante. Deve per forza essere abbinata alle altre classi di tipo `.btn-`. Quelli di Bootstrap l'avevano pensata così. 

Pertanto ho aggiornato il primo paragrafo di documentazione specificando meglio questo punto.
